content/deployment/extended_install.md

```diff
@@ -202,7 +202,7 @@ mv rules/RESPONSE-999-EXCLUSION-RULES-AFTER-CRS.conf.example rules/RESPONSE-999-
 
 The engine should support the `Include` directive out of the box. This directive tells the engine to parse *additional* files for directives. The question is where to put the CRS rules folder in order for it to be included.
 
-Looking at the CRS files, there are quite a few ".conf" files. While the names attempt to do a good job at describing what each file does, additional information is available in the [rules]({{< ref "../rules/" >}}) section.
+Looking at the CRS files, there are quite a few ".conf" files. While the names attempt to do a good job at describing what each file does, additional information is available in the [rules]({{% ref "rules" %}}) section.
```

Comment:
Does Hugo flatten the page tree suddenly? Or does it have "magic lookup"?

Reply:
This looks like a legacy magic miracle, when docs where in the same place as the rules. Clearly this won't work. But we can fix this in a followup, let me create an issue.

Reply:
Issue created.